charts/newrelic-logging/values.yaml

```diff
@@ -27,6 +27,7 @@
 fluentBit:
   logLevel: "info"
   path: "/var/log/containers/*.log"
+  linuxMountPath: /var
```

Comment:
I believe we don't need this 🤔

Reply:
@jsubirat - You mean the value `/var` or the variable itself?

Reply:
I meant the variable itself, but after your explanation about /var/lib you can ignore this comment